kuma/core/helpers.py

```diff
@@ -150,8 +151,11 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    if not locale:
+        locale = settings.LANGUAGE_CODE
+    filename = get_filename(locale, settings.STATICI18N_DOMAIN)
+    return static(os.path.join(settings.STATICI18N_OUTPUT_DIR, filename))
```

Comment:
Can we customize where the generated `.po` files are placed? If we can, it would be best to put them into *build/locale*. That way, `collectstatic` will ultimately move them into */static/locale* rather than moving each sub-directory into */static* directly.